irohad/ametsuchi/impl/rocksdb_common.hpp

```diff
@@ -244,6 +247,10 @@ namespace iroha::ametsuchi::fmtstrings {
    * ############# FOLDERS ################
    * ######################################
    */
+  // height ➡️ block data
+  static auto constexpr kBlockDataInStore{
+      FMT_STRING(RDB_ROOT /**/ RDB_STORE /**/ RDB_BLOCK_XXX)};
```

Comment:
You need sorted enumeration in iroha once in reindex WSV. So it is the single place you will add performacne with such a key. But you reduce it in a highly loaded queries(GetTransactions, GetAccountTransactions, GetAccountAssetTransactions) and syncronization because of such a long key. Revert plz to ```RDB_XXX```.

Reply:
If so and we will need such a enumeration, than we will make a solution with custom comparator.